ext/webextension/src/options/options.html

```diff
@@ -37,6 +37,11 @@
 input[type=checkbox] {
     width:1em;
 }
+#treat_as_same_site {
+    height: 15em;
+    white-space: nowrap;
```

Comment:
the nowrap is the problem.. appears that pre seems to be the correct(?) choice
ref https://bugzilla.mozilla.org/show_bug.cgi?id=1137650